docs/source/package_reference/hf_api.mdx

```diff
@@ -45,3 +45,10 @@ It does this using the `HfFolder` utility, which saves data at the root of the u
 
 [[autodoc]] HfFolder
 
+## Filtering helpers
+
+Some helpers to filter repositories on the hub are available in the `huggingface_hub` repository.
```

Comment:
I think in the documentation we should refer to `huggingface_hub` as a package, and not a repo. Users should be reading the docs and installing the package, so from their point of view the repo doesn't matter much.